cves/CVE-2014-3191.yml

```
@@ -154,38 +192,46 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies: 
+    applies: false
     note: 
   least_privilege:
-    applies: 
+    applies: false
     note: 
   frameworks_are_optional:
-    applies: 
-    note: 
+    applies: true
+    note: |
+      Vulnerability was found in Blink, a rendering engine Google forked from Webkit 
+      rather than continuing to use regular WebKit.
   native_wrappers:
-    applies: 
+    applies: false
     note: 
   distrust_input:
-    applies: 
+    applies: false
     note: 
   security_by_obscurity:
-    applies: 
+    applies: false
     note: 
   serial_killer:
-    applies: 
+    applies: false
     note: 
   environment_variables:
-    applies: 
+    applies: false
     note: 
   secure_by_default:
-    applies: 
+    applies: false
     note: 
   yagni:
-    applies: 
+    applies: false
     note: 
   complex_inputs:
-    applies: 
+    applies: false
     note: 
+  use_after_free:
+    applies: true
+    note: |
+      This vulnerability was similar to the double-free weakness discussed in class.  Instead of trying to free an already-freed
+      memory location, the code tried to make additional method calls with an object that had already been freed.  Like double-free,
+      the mitigation involves careful memory management and the weakness can easily be found by a fuzzer, as it was here.
```

Love this description, but take out the class reference. This is going to researches who may or may not have taken Meneely's class.
```suggestion
      This vulnerability is similar to a double-free weakness.  Instead of trying to free an already-freed
```